lib/schema.js

```diff
@@ -1,106 +1,100 @@
 // bucket schema
+const bucketStateKeys = {
+    storageUtilized: b => `s3:buckets:${b}:storageUtilized`,
+    numberOfObjects: b => `s3:buckets:${b}:numberOfObjects`,
+    incomingBytes: b => `s3:buckets:${b}:incomingBytes`,
+    outgoingBytes: b => `s3:buckets:${b}:outgoingBytes`,
+};
+
```

Comment:
A comment that these counters are to have unique ids for the range key value sets would be helpful.